### PR TITLE
Fix invalid denormalize result when query has multiple trees of same node

### PR DESCRIPTION
**Pull request by Jontem**

### CODE REVIEW

File: src/denormalize.ts [hunk 1]
```diff
@@ -105,7 +109,7 @@ export function denormalize(
           } else {
             // This field is a primitive (not a array or object)
             if (entityValue !== undefined) {
-              responseObjectOrNewParentArray[
+              (responseObjectOrNewParentArray as MutableResponseObject)[
                 (field.alias && field.alias.value) || field.name.value
```
Comment (jonaskello) [inline src/denormalize.ts:113]: `(field.alias && field.alias.value) || field.name.value` could perhaps be put into a const as it is used twice

**Comment by Jontem** [comment]
@jonaskello Please have a look

File: src/denormalize.ts (hunk 1 above)
Comment (jonaskello) [reply src/denormalize.ts:113]: I see now that it was fieldNode in the first place and field in the second. Nevermind :-)

**Review by jonaskello — APPROVED** [review]

**Comment by jonaskello** [comment]
LGTM

**Pull request merged**